core/mpi/routes/paciente.ts

```diff
@@ -597,6 +596,49 @@ router.post('/pacientes', (req, res, next) => {
     }
 });
 
+// Post para insertar un conjunto de pacientes contenidos en un array
+router.post('/pacientesAll', async (req, res, next) => {
+    if (!Auth.check(req, 'mpi:paciente:postAndes')) {
+        return next(403);
+    }
+    if (req.body.pacientesArray) {
+        let pacientes = req.body.pacientesArray;
+        let pacientesReturn = [];
+
+        for (let i = 0; i < pacientes.length; i++) {
+            if (pacientes[i].documento) {
+                const condicion = {
+                    documento: pacientes[i].documento
+                };
+                await controller.searchSimilar(paciente[i], 'andes', condicion).then(async (data) => {
```

Comment:
Unificar criterios. Están combinando estilo async/await con promesas.

```suggestion
    const data = await controller.searchSimilar(paciente[i], 'andes', condicion);
    if (data && data.length && data[0].value > 0.90) {
        ...
```

Tener en cuenta, que si encuentra algún paciente similar se ejecuta la función `next` de express, pero el programa sigue corriendo y la final hace `return res.json(pacientesReturn);`  produciendo un error.